rclpy/topics/pointcloud_publisher/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>pointcloud_publisher</name>
+  <version>0.0.1</version>
+  <description>Example on how to publish a Pointcloud2 message</description>
+  <maintainer email="evanflynn.msu@gmail.com">Evan Flynn</maintainer>
+  <license>Apache 2.0</license>
+
+  <depend>sensor_msgs</depend>
+  <depend>std_msgs</depend>
```

Comment:
Since this is a python package, these can just be `exec_depend`.

You need an `exec_depend` on `rclpy`, `sensor_msgs_py`, and `python3-numpy` as well.